### PR TITLE
Added TLS config to readme

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -504,6 +504,56 @@ class StaticBitIterator implements BitIterator {
 }
 ```
 
+## SSL/TLS
+
+Make sure the Pilosa server runs on a TLS address. [How To Set Up a Secure Cluster](https://www.pilosa.com/docs/latest/tutorials/#how-to-set-up-a-secure-cluster) tutorial explains how to do that.
+
+In order to enable TLS support on the client side, scheme of the address should be `https` which should be explicitly specified, e.g.: `https://01.pilosa.local:10501`
```
Comment (travisturner) [inline README.md]: "... client side, the scheme of the address should be explicity specified as `https`, e.g.: ..."

File: README.md [hunk 2]
```diff
@@ -504,6 +504,56 @@ class StaticBitIterator implements BitIterator {
...
+
+This client library uses the [Apache HTTP Library](https://hc.apache.org). `ClientOptions` builder accepts an `javax.net.ssl.SSLContext` object, which is set to `org.apache.http.ssl.SSLContexts.createDefault()` by default. If the Pilosa server is using a certificate from a recognized authority, you can use the defaults.
```
Comment (travisturner) [inline README.md]: accepts a (not "an")

**Comment by yuce** [comment]
@travisturner updated

File: README.md [hunk 3]
```diff
@@ -504,6 +504,56 @@ class StaticBitIterator implements BitIterator {
 }
 ```
 
+## SSL/TLS
+
+Make sure the Pilosa server runs on a TLS address. [How To Set Up a Secure Cluster](https://www.pilosa.com/docs/latest/tutorials/#how-to-set-up-a-secure-cluster) tutorial explains how to do that.
+
+In order to enable TLS support on the client side, the scheme of the address should be explicity specified as https, e.g.: `https://01.pilosa.local:10501`
```
Comment (travisturner) [inline README.md]: did you want backticks around `https` here? just asking because you removed them.

Comment (yuce) [reply README.md]: Sorry, I've just seen this comment. `https` lost its backticks on copy/paste. I've put it back.

**Pull request merged**